sql/wsrep_mysqld.cc

```diff
@@ -688,6 +688,7 @@ void wsrep_deinit()
 
 void wsrep_recover()
 {
+  #if 0
```

Comment:
Seems like you cooked up this patch over and above the existing branch. Each JIRA issue should have its own separate branch created from trunk. Check the guideline page on wiki.